docs/integration.md

```diff
@@ -4,87 +4,128 @@
 - [Error solutions](#error-solutions)
 - [Additional info](#additional-info)
 
-## Simple steps
+## In simple steps
 
-1. Ensure you have satisfied [building dependencies](building.md)
+1. Ensure you have satisfied the [building dependencies](building.md)
 
-2. To use eSpeak NG as a C library, create file `test-espeak.c` with following content:
+2. To use eSpeak NG as a C library, create a file called `test-espeak.c` with the following content:
 
 
-        #include <espeak-ng/speak_lib.h>
-        
-        espeak_AUDIO_OUTPUT output = AUDIO_OUTPUT_SYNCH_PLAYBACK;
-        char *path = NULL;
-        void* user_data;
-        unsigned int *identifier;
-        
-        int main(int argc, char* argv[]) {
-          char voicename[] = {"English"}; // Set voice by its name
-          char text[] = {"Hello world!"};
-          int buflength = 500, options = 0;
-          unsigned int position = 0, position_type = 0, end_position = 0, flags = espeakCHARS_AUTO;
-          espeak_Initialize(output, buflength, path, options);
-          espeak_SetVoiceByName(voicename);
-          printf("Saying  '%s'...\n", text);
-          espeak_Synth(text, buflength, position, position_type, end_position, flags, identifier, user_data);
-          printf("Done\n");
-          return 0;
-        }
+    #include <espeak-ng/speak_lib.h>
+    
+    espeak_AUDIO_OUTPUT output = AUDIO_OUTPUT_SYNCH_PLAYBACK;
+    char  *path = NULL;
+    void  *user_data;
+    unsigned int *identifier;
+    
+    int
+    main(int argc, char *argv[])
+    {
+            char *voicename = "English";            /* Set voice by its name */
+            char *text      = "Hello world!";
+            int   buflength = 500,
+            int   options   = 0;
+
+            unsigned int position      = 0,
+                         position_type = 0,
```

Comment:
Why these are indented more?

Reply:
Placing multiple declarations/assignments on a single line is advised when the variables are few and closely related. Doing so with more than 4 and ones with long names reduces overall readability of that fragment of the code. Moreover, the type of the variables, `unsigned int` is rather verbose. It would be preferable to change that type to `int` if the interface allows.
This could be (partially) mitigated by changing the variables prefix to `pos`